include/constants/pokemon_config.h

```diff
@@ -18,6 +18,8 @@
 #define P_UPDATED_EGG_GROUPS    GEN_8 // Since Gen 8, certain Pokémon have gained new egg groups.
 #define P_SHEDINJA_BALL         GEN_8 // Since Gen 4, Shedinja requires a Poké Ball for its evolution. In Gen 3, Shedinja inherits Nincada's Ball.
 
+#define P_ENABLE_DEBUG                // Enables a debug menu for pokemon sprites and icons, accessed by pressing SELECT in the summary screen.
+
 #ifndef ITEM_EXPANSION
```

Comment:
```suggestion
#define P_SHEDINJA_BALL         GEN_8 // Since Gen 4, Shedinja requires a Poké Ball for its evolution. In Gen 3, Shedinja inherits Nincada's Ball.

#define P_ENABLE_DEBUG           TRUE // If set to TRUE, enables a debug menu to use for pokemon sprites and icons in the summary screen accessed by pressing SELECT.
```

To remain consistent with the BE's.
For maximum consistency, it should be moved to the end of the file, just before the guard constants.

Reply:
I feel the one in BE should be moved up. It's a pretty big feature that some people might miss because it's so far down.

Reply:
That works too :eyes: